Adjust heroku (#6)
CorndelWithSoftwire/DevOps-Course-Workshop-Module-08-Learners: during_workshop_8.md

4. You should now see a log of the deployment on your Heroku app's dashboard: https://dashboard.heroku.com/apps/<HEROKU_APP_NAME> (replace <HEROKU_APP_NAME> with the name you gave your Heroku app when you created it).
5. You can see the app running by clicking the "Open app" button on the app's dashboard, or by going to <HEROKU_APP_NAME>.herokuapp.com (replace <HEROKU_APP_NAME> with the name you gave your Heroku app when you created it).

### Deploy to Heroku with GitHub Actions
Add a new step to your workflow which will deploy to Heroku. You should be able to find an existing action to do this for you. As with the publish step, make sure this only runs on the main branch.

<details>
<summary>Hint</summary>

You might want to look [at this action](https://github.com/marketplace/actions/deploy-to-heroku).

<details>
<summary>Hint</summary>

> See the example ["Deploy with Docker"](https://github.com/marketplace/actions/deploy-to-heroku#deploy-with-docker) section, and don't forget the `useddocker` flag

Reviewer: Minor typo
```suggestion
See the example ["Deploy with Docker"](https://github.com/marketplace/actions/deploy-to-heroku#deploy-with-docker) section, and don't forget the `usedocker` flag
```

</details>
</details>

###  (Stretch goal) Multistage Dockerfile

> You may have noticed that our Dockerfile is pretty sizeable (~1.5GB) and apart from taking up space, it also slows our pipeline down as the upload step tends to take a while. .NET allows us to separate the dependencies we need for building the code to those that we need for running the compiled binary with the latter being quite a bit smaller. This offers a good opportunity here to optimise the speed of our deployment pipeline.

Reviewer: Rephrased this slightly to add a little more detail:
```suggestion
You may have noticed that the image our Dockerfile builds is pretty sizeable (~1.5GB) and, apart from taking up space, it slows our pipeline down during the upload step. .NET allows us to separate the dependencies needed to build the code (part of the SDK) from those needed to run the compiled binary (the runtime), with the latter being much smaller. This offers a good opportunity to optimise the speed of our deployment pipeline.
```

> Try writing your Dockerfile as a multistage build. For [an example that closely matches this project see here](https://github.com/dotnet/dotnet-docker/blob/main/samples/aspnetapp/Dockerfile) - or [see the Docker docs](](https://docs.docker.com/samples/dotnetcore/#create-a-dockerfile-for-an-aspnet-core-application)) for another approach.

Reviewer: Fixes broken link:
```suggestion
Try writing your Dockerfile as a multistage build. For [an example that closely matches this project see here](https://github.com/dotnet/dotnet-docker/blob/main/samples/aspnetapp/Dockerfile) - or [see the Docker docs](https://docs.docker.com/samples/dotnetcore/#create-a-dockerfile-for-an-aspnet-core-application) for another approach.
```

> To make the example work:

Reviewer: ```suggestion
To make the first example linked above work:
```
